scripts/jestPerTestSetup.ts

```diff
@@ -0,0 +1,75 @@
+import fs from 'fs-extra'
+import { resolve } from 'path'
+import { FastifyInstance } from 'fastify'
+import { Page } from 'playwright-chromium'
+
+export function slash(p: string): string {
+  return p.replace(/\\/g, '/')
+}
+
+// injected by the test env
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
+  namespace NodeJS {
+    interface Global {
+      page?: Page
+      fastifyRendererTestUrl?: string
+    }
+  }
+}
+
+let server: FastifyInstance
+let err: Error
+
+const logs: string[] = ((global as any).browserLogs = [])
```

Comment:
I guess this is for asserting against which we arent using yet but might?

Reply:
Exactly, will remove it if I don't end up using it but I thought it would be handy to assert against and make sure there were no console errors.